judgments/views/detail.py

```diff
@@ -79,23 +84,26 @@ def detail(request, document_uri):
     context = {}
 
     press_summary_suffix = "/press-summary/1"
-    if document_uri.endswith(press_summary_suffix):
-        context["document_type"] = "press_summary"
+    context["document_noun"] = document.document_noun
+    if document.best_human_identifier is None:
+        raise NoNeutralCitationError(document.uri)
```

Comment:
I think we should never reach the scenario of NoNeutralCitationError in a published document but this will make it easier for us to debug any edge cases that slip through for whatever reason.

Could we add tests for when we raise this then:

i.e for line 89 and 106.



even though we should not reach this scenario can we add a test for this scenario